dev/test/specs/starter-test.js

```diff
@@ -231,8 +248,28 @@ describe('test basic arcs functionality', function() {
     acceptSuggestion(footerPath, 'make a reservation');
     acceptSuggestion(footerPath, 'You are free');
 
+    browser.close();
     // to drop into debug mode with a REPL; also a handy way to see the state
     // at the end of the test:
     // browser.debug();
   });
+
+  it('can use the gift shopping demo flow', function() {
```

Comment:
This method looks pretty good! It seems pretty literate. We could remove the footerPath argument (maybe?) to clean it up a little further, even.

Reply:
done!